modules/ckanext-sixodp_showcase/ckanext/sixodp_showcase/commands.py

```diff
@@ -90,3 +91,36 @@ def migrate_title_to_title_translated(ctx, config, dryrun):
     else:
         # No resource patches so empty parameter is passed
         apply_patches(showcase_patches, [])
+
+
+@sixodp_showcase_group.command(
+    u'create_platform_vocabulary',
+    help=u'Creates a platforms vocabulary to use as a preset list of options'
+)
+@click_config_option
+@click.option(u'--dryrun', is_flag=True)
```

Comment:
There is a dryrun flag, but it isn't used. Either remove the flag or implement functionality for it.

Reply:
Implemented dryrun functionality in 4d03743